learning/courses/foundations-of-quantum-error-correction/quantum-code-constructions/toric-code.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0e334530",
+   "metadata": {},
+   "source": [
+    "{/* cspell:ignore LDPC */}\n",
+    "{/* cspell:ignore tensoring */}\n",
+    "\n",
+    "# Toric code\n",
+    "\n",
+    "Next we'll discuss a specific CSS code known as the *toric code*, which was discovered by Alexei Kitaev in 1997.\n",
+    "In fact, the toric code isn't a single code, but rather it's a family of codes, one for each positive integer at least 2.\n",
```

Comment:
should this be "of at least 2"? @jhwatrous

Reply:
"... each positive integer starting from 2" or "... each positive integer that is at least 2" perhaps? I see that what I wrote is awkward, but I'm not sure adding "of" is right.

Reply:
I'll use "starting from 2" - I think that's clearer. Thanks!